pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -63,16 +29,6 @@ def upsample() -> nn.Upsample:
     return nn.Upsample(scale_factor=2.0, mode="nearest")
 
 
-class HourGlassBlock(SequentialWithOutChannels):
```

Comment:
On a second thought, we might just leave this here. My initial thought was that this is a commonly used structure, but with the current implementation I think its a bit too generic. Right now its basically an `nn.Sequential` with the three components. Thoughts?